suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -709,6 +709,12 @@ export const selectDeviceByState = (state: DeviceRootState, deviceState: string)
 export const selectDeviceUnavailableCapabilities = (state: DeviceRootState) =>
     state.device.selectedDevice?.unavailableCapabilities;
 
+export const selectDeviceCapabilities = (state: DeviceRootState) =>
+    selectDeviceFeatures(state)?.capabilities;
+
+export const selectHasDevicePassphraseEntryCapability = (state: DeviceRootState) =>
```

Comment:
```suggestion
export const selectDeviceHasPassphraseEntryCapability = (state: DeviceRootState) =>
```

I think this one is correct. But I'm not sure. Leaving it as a nitpick, nothing critical

Reply:
If you change it, please also change variable name in component as well

Reply:
I think this form is better due to clarity of what is the selector result and it is in line with similar selectors that are also returning boolean like:

`selectIsDeviceBitcoinOnly`
`selectHasDeviceFirmwareInstalled`
`selectIsDeviceConnected`
`selectIsDeviceInViewOnlyMode`

I thought about the form you are suggesting before, but this one feels more spot on to me.